way_to_home/utils/file_handlers.py

```diff
@@ -0,0 +1,87 @@
+"""This module implements helpers functions for work with files."""
+
+import pickle
+import csv
+from zipfile import ZipFile
+import requests
+
+
+def _get_fields_position(row_header, required_fields):
+    """
+    Return dictionary where the keys are required fields and
+    values are numbers of columns with the appropriate name.
+    """
+    fields_position = {}
+    for field in required_fields:
+        fields_position[field] = row_header.index(field)
+
+    return fields_position
+
+
+def unzip_file(path_to_file, unzip_to='./'):
+    """Unzip file to directory with path `unzip_to`."""
+    try:
+        with ZipFile(path_to_file, 'r') as zip_file:
+            zip_file.extractall(unzip_to)
+    except FileNotFoundError:
+        return False
+
+    return True
+
+
+def load_file(url, save_to='./'):
```

Comment:
where you use this function?

Reply:
It is helpers function. Celery will use this function for download files from Easy Way. Also this function can be used for downloading other files if needed.